src/e2e/testUtils.ts

```diff
@@ -14,9 +14,9 @@ const delay = (ms) =>
  *
  * @returns {Promise} The promise from deleting messages
  */
-export const trashChat = () =>
-  game.messages.size > 0
-    ? game.messages.documentClass.deleteDocuments([], { deleteAll: true })
+export const trashChat = (): Promise<any> =>
+  game.messages?.size > 0
```

Comment:
Since we're using a question mark here, maybe we ought to check for if `game.messages?.size` is truthy, instead of checking for if it's greater than 0.

Reply:
I'm not sure why this question-mark is there, as this should always be defined. I'll fix this.

Reply:
Makes sense, I think I put the question mark there as I got warnings that it may be empty. But since we are running tests with Foundry initiated, I think we can just do a straight check here.